File: MobileApp/app/src/main/java/com/guanacobusiness/model/inscription.kt
package com.guanacobusiness.model

import com.google.gson.annotations.SerializedName

data class inscription(
    @SerializedName("_id")val id: String,
    @SerializedName("job")val job: job,
    @SerializedName("applicant")val applicant: user,
    @SerializedName("employer")val employer: user
)

File: APIGuanacoBusiness/src/models/inscription.js
const mongoose = require("mongoose");

const Schema = mongoose.Schema;
const InscriptionSchema = new Schema(
  {
    job: { 
        type: Schema.Types.ObjectId, 
        ref: "job", 
        required: true },
    applicant : {
        type: Schema.Types.ObjectId,
        ref: "user",
        required: true
    },
    employer : {
        type: Schema.Types.ObjectId,
        ref: "user",
        required: true
    }   
  },
  { timestamps: true }
);

const InscriptionModel = mongoose.model("inscription", InscriptionSchema);

module.exports = InscriptionModel;